src/Microsoft.AspNetCore.Mvc.RazorPages/PageModel.cs

```diff
@@ -1614,6 +1615,27 @@ public virtual ObjectResult StatusCode(int statusCode, object value)
         public virtual UnauthorizedResult Unauthorized()
             => new UnauthorizedResult();
 
+        /// <summary>
+        /// Creates a <see cref="PartialViewResult"/> by specifying the name of a partial to render.
+        /// </summary>
+        /// <param name="partialViewName">The partial name.</param>
+        /// <returns>The created <see cref="PartialViewResult"/> object for the response.</returns>
+        public virtual PartialViewResult Partial(string partialViewName)
+            => Partial(partialViewName, model: null);
+
+        /// <summary>
+        /// Creates a <see cref="PartialViewResult"/> by specifying the name of a partial to render and the model object.
+        /// </summary>
+        /// <param name="partialViewName">The partial name.</param>
+        /// <param name="model">The model to be passed into the partial.</param>
+        /// <returns>The created <see cref="PartialViewResult"/> object for the response.</returns>
+        public virtual PartialViewResult Partial(string partialViewName, object model)
+            => new PartialViewResult
```

Comment:
Could you use a regular function here? The rule of thumb we've been using is to use lambdas for one-lines, regular functions otherwise.

Reply:
It's one line!!